ddd-kata.md

```diff
@@ -0,0 +1,197 @@
+| [Bounded Context Canvas](https://github.com/ddd-crew/bounded-context-canvas)     | The Bounded Context Canvas is a collaborative design tool for creating and documenting a single bounded context. The canvas walks you through the process of developing a bounded context by pushing you to examine and make decisions about essential design components such as nomenclature, responsibilities, public interface, and dependencies. Use the Bounded Context Canvas to model and understand the scope of the Bounded Context itself.                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| [Aggregate Canvas](https://github.com/ddd-crew/aggregate-design-canvas)          | The Aggregate Design Canvas is a modeling tool that should be used in conjunction with design-level domain modeling efforts. Eric Evans first described aggregates as a lifecycle pattern. We define aggregate as a network of objects that serves as a consistency boundary for our domain regulations. Depending on how the aggregate is designed, we can either enforce them (making them invariant) or be compelled to implement corrective measures. As a result, it is critical to carefully establish aggregate borders, as they influence the behaviors modeled inside our domain. The canvas features a suggested working order that aids in iteratively discussing various components of the overall design. Here you go back to the Event Storming and find the aggregates defined in the process. Aggregates with the same name can appear in several bounded contexts. They are different and might reflect different aspects of the process.                                                   |
+
+## The DDD Kata
+
+### Task
+
+* Select a collaboration tool for your choice. In SAP we are using Mural. In the DDD Community Miro is used a lot.
+* **Form a working group** (up to 8 people). Declare one member a domain expert (maybe a substitute) and one member as facilitator / moderator
+* Perform a **Big Picture Event Storming** with your group in collaboration tool of choice
+* **Identify the bounded context and aggregates** within evaluated process in collaboration tool of choice
+* Establish the message flow from Bounded Context to Bounded Context using the **Domain Message Flow** Diagrams in collaboration tool of choice.
+* For each identified Bounded Context, specify using the **Bounded Context Canvas** in collaboration tool of choice.
+* Within the defined Bounded Contexts, defined the aggregates using the **Aggregate canvases** in collaboration tool of choice.
```

Comment:
All of the sentences now have "in collaboration tool of choice" at the end. Personally, I'd remove it completely. If you want to keep it, I'd change it to "in _your_ collaboration [...]".